src/facts/stiff-leaved-goldenrod-1.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Stiff-leaved Goldenrod
+image:
+  src: ../images/stiff-leavedgoldenrod.jpg
+  alt: Stiff-leaved Goldenrod
+  creator: Andrew Cannizzaro
+  license: CC by 2.0
+  url: https://www.flickr.com/photos/acryptozoo
+source:
+  url: https://www.pollinatorsnativeplants.com/bees-an-identification-and-native-plant-forage-guide.html
+  name: "Bees: An Identification and Native Plant Forage Guide"
+category: Habitat
+---
+<!--StartFragment-->
+
+**This species is more commonly visited by female bees than male, due to it flowering earlier than other goldenrod species (bee book,p. 185)**
```

Comment:
Be careful just pasting in from the document

Reply:
It is better to right click and "paste as plain text" and don't include the "(bee book,p. 185)". This shouldn't be bold.